plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateways/test/payment-gateways.test.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { recordEvent } from '@woocommerce/tracks';
+import { render, fireEvent, waitFor } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { PaymentGateways } from '..';
+
+jest.mock( '@woocommerce/tracks', () => ( {
+	recordEvent: jest.fn(),
+} ) );
+
+describe( 'PaymentGateways', () => {
+	it( 'should fire settings_payments_business_location_update	when country selection is changed', async () => {
+		const setBusinessRegistrationCountry = jest.fn();
+		const { getByRole, findByText } = render(
+			<PaymentGateways
+				providers={ [] }
+				installedPluginSlugs={ [] }
+				installingPlugin={ null }
+				setupPlugin={ jest.fn() }
+				acceptIncentive={ jest.fn() }
+				updateOrdering={ jest.fn() }
+				isFetching={ false }
+				businessRegistrationCountry={ null }
+				setBusinessRegistrationCountry={
+					setBusinessRegistrationCountry
+				}
+			/>
+		);
+
+		const countrySelectorButton = getByRole(
+			'combobox'
+		) as HTMLSelectElement;
+		fireEvent.click( countrySelectorButton );
+		const ukraineOption = await findByText( 'Ukraine' );
```

Comment:
This fails because country selector component is rendered in the test env. with an empty list.